bots/bots/aggregation-bot.js

```diff
@@ -13,29 +17,53 @@ class AggregationBot {
    */
   async main() {
     const measurables = await this.api.measurables();
+    if (measurables.length === 0) {
+      console.log('Measurables list is empty.');
+      return true;
+    }
+
+    console.log(`Going to process ${measurables.length} measurables.`);
 
     for (const measurable of measurables) {
-      const measurableId = measurable.id;
-      const measurements = await this.api.measurementsCompetitive({
-        measurableId,
-      });
+      const id = { measurableId: measurable.id };
+
+      const measurements = await this.api.measurementsCompetitive(id);
+      console.log(`Going to process ${measurements.length} measurements.`);
+      if (measurements.length === 0) continue;
+
       const aggregated = await this.aggregate(measurements);
-      await this.api.measurementCreateAggregation({
-        measurableId,
-        ...aggregated,
-      });
+      if (!aggregated) continue;
+
+      await this.api.measurementCreateAggregation({ ...id, ...aggregated });
     }
 
     return true;
   }
 
   /**
+   * @todo: What to do with "floatPoint" spot?
```

Comment:
points should never be aggregated. Points can only be final values, not competitive measuremenets.

Reply:
ok